pkg/smokescreen/smokescreen.go

```diff
@@ -642,8 +642,11 @@ func handleConnect(config *Config, pctx *goproxy.ProxyCtx) (string, error) {
 		return "", pctx.Error
 	}
 
-	// checkIfRequestShouldBeProxied can return an error if either the resolved address is disallowed,
-	// or if there is a DNS resolution failure.
+	/*
```

Comment:
Just for consistency and since this is only 3 lines could we keep using the single line comment format?

Reply:
Sure, fixed